grammar.js

```diff
@@ -463,9 +462,12 @@ module.exports = grammar({
       $.set_comprehension,
       $.tuple,
       $.generator_expression,
-      $.ellipsis
+      $.ellipsis,
+      $.empty_expression
     ),
 
+    empty_expression: $ => seq('(',')'),
```

Comment:
Prior to this, we would match `()` as a `tuple`. By adding `empty_expression` to `_expression`, we can now match against `()` without mislabelling as a `tuple`, yet, this name may not be the best. @maxbrunsfeld if you have thoughts or if there is a convention found in other tree-sitter grammars that would work well here, I'm open to suggestions!